x-pack/test/functional/page_objects/gis_page.ts

```diff
@@ -294,6 +294,40 @@ export class GisPageObject extends FtrService {
     await this.testSubjects.click('layerVisibilityToggleButton');
   }
 
+  async toggleLayerVisibilityRoadMap() {
```

Comment:
Can you move toggleLayerVisibilityRoadMap and toggleLayerVisibilityTotalRequests out of gis_page object and into maps_smoke_tests since they are specific to the upgrade testing and not general to all testing.

After the move can you add comments on why there are 2 things to check, something like

```
// In v8.0, the default base map switched from bright to desaturated.
// https://github.com/elastic/kibana/pull/116179 
// Maps created before this change will have a base map called "Road map"
// Maps created after this change will have a base map called "Road map - desaturated" 
// toggleLayerVisibilityRoadMap will toggle layer visibility for either value
async toggleLayerVisibilityRoadMap() {...}

// In v7.16, e-commerce sample data was re-worked so that geo.src field to match country code of geo.coordinates
// https://github.com/elastic/kibana/pull/110885
// Maps created before this change will have a layer called "Total Requests by Country"
// Maps created after this change will have a layer called "Total Requests by Destination" 
// toggleLayerVisibilityTotalRequests will toggle layer visibility for either value
async toggleLayerVisibilityTotalRequests() {...}
```

Reply:
Thanks @nreese done.